keras/tests/integration_test.py

```diff
@@ -317,5 +317,52 @@ def test_serialization_v2_model(self):
     loaded_model = keras.models.load_model(output_path)
     self.assertEqual(model.summary(), loaded_model.summary())
 
+@test_combinations.run_with_all_model_types
+@test_combinations.run_all_keras_modes
```

Comment:
You may want to avoid running in V1 mode

Reply:
Done!